cmd/components_check.go

```diff
@@ -64,7 +64,7 @@ and validating the health field:
 }
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		RunCheck(NewComponentCheck("DC/OS components health check"))
+		RunCheck(nil, NewComponentCheck("DC/OS components health check"))
```

Comment:
Should we be passing `context.Background()` here instead of `nil` for the context argument?